docs/cli/ignite/ignite_image_import.md

```diff
@@ -23,11 +23,9 @@ ignite image import <OCI image> [flags]
 ### Options inherited from parent commands
 
 ```
-      --ignite-config string    Ignite configuration path; refer to the 'Ignite Configuration' docs for more details
-      --log-level loglevel      Specify the loglevel for the program (default info)
-      --network-plugin plugin   Network plugin to use. Available options are: [cni docker-bridge] (default cni)
-  -q, --quiet                   The quiet mode allows for machine-parsable output by printing only IDs
-      --runtime runtime         Container runtime to use. Available options are: [docker containerd] (default containerd)
```

Comment:
image import should probably have a runtime flag.

does not need to read any vm object.

flag is simply present to override ignite global config.